src/modules/vendor/components/programs/cards/templates/MultipleChoiceQuestion.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div v-if="answersInitialized">
+    <ni-input caption="Question" v-model.trim="card.question" required-field @focus="saveTmp('question')"
+      @blur="updateCard('question')" :error="$v.card.question.$error" type="textarea" />
+    <div class="q-my-xl">
+      <div v-for="(qcmAnswers, i) in card.qcmAnswers" :key="i" class="answers">
+        <ni-input :caption="`Réponse ${i + 1}`" v-model.trim="card.qcmAnswers[i].label" :required-field="i < 2"
+          @focus="saveTmp(`qcmAnswers[${i}].label`)" @blur="updateAnswer(i)"
+          :error="answersError(i)" :error-message="answersErrorMsg(i)" />
+        <q-checkbox v-model="correctAnswerIndexes[i]" @input="updateCorrectAnswer(i)"
+          :disable="!card.qcmAnswers[i].label" />
+      </div>
+    </div>
+    <ni-input caption="Correction" v-model.trim="card.explanation" required-field @focus="saveTmp('explanation')"
+      @blur="updateCard('explanation')" :error="$v.card.explanation.$error" type="textarea" />
+  </div>
+</template>
+
+<script>
+import times from 'lodash/times';
+import get from 'lodash/get';
+import Cards from '@api/Cards';
+import Input from '@components/form/Input';
+import { templateMixin } from 'src/modules/vendor/mixins/templateMixin';
+import { NotifyNegative, NotifyPositive, NotifyWarning } from '@components/popup/notify';
+import { MULTIPLE_CHOICE_QUESTION_MAX_ANSWERS_COUNT, REQUIRED_LABEL } from '@data/constants';
+
+export default {
+  name: 'MultipleChoiceQuestion',
+  components: {
+    'ni-input': Input,
+  },
+  mixins: [templateMixin],
+  data () {
+    return {
+      correctAnswerIndexes: [true, false, false, false],
+      answersLengthInDb: 0,
+    };
+  },
+  computed: {
+    answersInitialized () {
+      return this.card.qcmAnswers.length === MULTIPLE_CHOICE_QUESTION_MAX_ANSWERS_COUNT;
+    },
+  },
+  async mounted () {
+    this.initializeAnswers();
+    this.updateCorrectAnswerIndexes();
+  },
+  watch: {
+    card () {
+      this.initializeAnswers();
+      this.updateCorrectAnswerIndexes();
+    },
+  },
+  methods: {
+    initializeAnswers () {
+      this.answersLengthInDb = this.card.qcmAnswers.length;
+      this.card.qcmAnswers = times(
+        MULTIPLE_CHOICE_QUESTION_MAX_ANSWERS_COUNT,
+        i => this.card.qcmAnswers[i] || ({ label: '', correct: i === 0 })
```

Comment:
`correct: i === 0`  attention la tu modifies les données - si la carte que tu recois a la bonne reponse ne 2 tu ecarses cette info ...

Reply:
si elle a une bonne réponse, elle a une réponse associée donc la valeur sera : this.card.qcmAnswers[i], non ? Je n'arrive pas à reproduire ton cas

Reply:
en supprimant le tableau `correctAnswers` ca a corrigé le souci